.gitignore

```diff
@@ -0,0 +1,14 @@
+# maven
```

Comment:
Can you remove this please and rather use a local gitignore

Reply:
done